### PR TITLE
Update Alien Example to use for loop instead of for-each ... so it can be updated in-place.

### LINKED ISSUE #62: Alien translator: modify a collection in a foreach loop
The last code block of the Alien translator notebook lacks clarity.

Since you use the same collection variable before and after the foreach loop, it seems like you want the "student" to modify the collection in-place, which is not possible using a foreach loop.

Since the notebook is aimed at new C# programmers, it might be better to hint at a clearer pattern.

Maybe I'm missing something?

### PR DESCRIPTION
**Pull request by JakeRadMSFT**
Update Alien Example to use for loop instead of for-each ... so words array can be updated in-place.

@katiesavage @jamesmontemagno  Is this how it was intended to be?

Resolves #62

You can view the updated Notebook here: 
https://github.com/dotnet/csharp-notebooks/blob/32bc3eb9f1a2535d212506393e0a2dc561974693/csharp-scenarios/01-Alien-Translator.ipynb

### CODE REVIEW

File: csharp-scenarios/01-Alien-Translator.ipynb [hunk 1]
```diff
@@ -1,484 +1,477 @@
...
+      "metadata": {},
+      "source": [
+        "![Split Method Image](https://raw.githubusercontent.com/katiesavage/csharp-notebooks/main/csharp-scenarios/images/split-method.png)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Now that we can split our sentence string into word substrings, it’s time to loop over them and translate! To do this, we’ll use a [foreach loop](https://docs.microsoft.com/dotnet/csharp/language-reference/statements/iteration-statements#the-foreach-statement). This is a special type of loop that executes a statement for each element in a group of that type of element. \n",
```
Comment (katiesavage) [inline csharp-scenarios/01-Alien-Translator.ipynb:373]:
```suggestion
        "Now that we can split our sentence string into word substrings, it’s time to loop over them and translate! To do this, we’ll use a [for loop](https://docs.microsoft.com/dotnet/csharp/language-reference/statements/iteration-statements#the-for-statement). This is a special type of loop that executes a statement for a specified number of times. \n",
```

File: csharp-scenarios/01-Alien-Translator.ipynb [hunk 2]
```diff
@@ -1,484 +1,477 @@
...
+        "var sentence = \"this is an example\";\n",
+        "\n",
+        "// split the string into substrings\n",
+        "string[] variableCollection = sentence.Split(' ');\n",
+        "\n",
+        "foreach(var variableName in variableCollection) \n",
+        "{ \n",
+        "     // statements to be executed \n",
+        "     Console.Write(variableName + \" \");\n",
+        "} "
```
Comment (katiesavage) [inline csharp-scenarios/01-Alien-Translator.ipynb:396]:
```suggestion
        "for(var i = 0; i < variableCollection.Length; i++) \n",
        "{ \n",
        "     // statements to be executed \n",
        "     Console.Write(variableCollection[i] + \" \");\n",
        "} "
```

**Review by katiesavage — COMMENTED** [review]
Switched the above explanation & example to use a for loop.

File: csharp-scenarios/01-Alien-Translator.ipynb (hunk 1 above)
Comment (JakeRadMSFT) [reply csharp-scenarios/01-Alien-Translator.ipynb]: Merged in thanks!

File: csharp-scenarios/01-Alien-Translator.ipynb (hunk 2 above)
Comment (JakeRadMSFT) [reply csharp-scenarios/01-Alien-Translator.ipynb]: Merged in - thanks!

**Pull request merged**